test/demo/utils/integrationSetupAfterEnv.ts

```diff
@@ -0,0 +1,2 @@
+// Apps are taking a very long time to spawn on CI causing timeouts
+jest.setTimeout(60 * 1000);
```

Comment:
This is true currently, but we should look to remove/shorten this timeout once we have a fix for those issues.

Can we put a placeholder/reminder story in the backlog, and tag this with a "TODO" to clean this up?

Reply:
Sure thing!

Reply:
Actually added the TODO this time :P